otter/models/cass.py

```diff
@@ -1435,7 +1372,7 @@ def get_counts(self, log, tenant_id):
         fields = ['scaling_group', 'scaling_policies', 'policy_webhooks']
         deferred = [self.connection.execute(_cql_count_for_tenant.format(cf=field),
                                             {'tenantId': tenant_id},
-                                            self.get_consistency('count', 'group'))
+                                            DEFAULT_CONSISTENCY)
```

Comment:
This can be ONE since this operation can be expensive and we don't need to be accurate. However this function is not used anywhere.